peachjam_ml/apps.py

```diff
@@ -7,3 +7,8 @@ class PeachjamMLConfig(AppConfig):
 
     def ready(self):
         import peachjam_ml.signals  # noqa
+        from peachjam.views import BaseDocumentDetailView
+
+        from .handler import modify_document_detail_context
+
+        BaseDocumentDetailView.modify_context.connect(modify_document_detail_context)
```

Comment:
this is fine for now, but usually this would be connected in signals.py with the usual decorators